packages/ui/src/components/List/list.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useMemo } from 'react'
+import { List as DefaultList, ListProps as DefaultListProps } from 'antd'
+import classNames from 'classnames'
+import { Typography, TypographyTextProps } from '@open-condo/ui/src'
+import { ListItemProps as DefaultListItemProps } from 'antd/es/list'
```

Comment:
third-party import should be above first party...
and why from `/es` instead of `/lib`